flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamGraphGeneratorBatchExecutionTest.java

```diff
@@ -491,6 +494,55 @@ public void testCoFeedbackThrowsExceptionInBatch() {
         testNoSupportForIterationsInBatchHelper(coFeedbackTransformation);
     }
 
+    @Test
+    public void operatorSortWithoutRepetitionTest() {
```

Comment:
Typically we expect the test method name to start with `test...` and shows on the key logic/API that this test covers.

How about naming it something like `testInternalSorterAndOutputOnEOF`?